runtime/doc/usr_25.txt

```diff
@@ -248,18 +249,15 @@ When used on the second line of the example text, this is what you get:
 "4>>" will increase the indent of four lines.
 
 
-TABSTOP
+SMART TAB
 
 If you want to make indents a multiple of 4, you set 'shiftwidth' to 4.  But
-when pressing a <Tab> you still get 8 spaces worth of indent.  To change this,
-set the 'softtabstop' option: >
+when pressing a <Tab> you still get 8 columns worth of indent.
+To change this, set the 'smarttab' option. >
 
-	:set softtabstop=4
+	:set smarttab
 
-This will make the <Tab> key insert 4 spaces worth of indent.  If there are
-already four spaces, a <Tab> character is used (saving seven characters in the
-file).  (If you always want spaces and no tab characters, set the 'expandtab'
-option.)
+This will make the <Tab> key indent by 4 columns.
```

Comment:
I don't like this block. It's true that smartab achieves that behaviour, but I'd prefer if we keep the existing documentation, because:
1) I believe the usage of smarttab has been largely deprecated in favor of the softtabstop option and I don't want to recommend to new users to use the smarttab setting
2) you would only get an indent of 4 columns if you are in leading whitespace, but not after you have written some text. I find that highly confusing and would recommend to use the softtabstop option for consistency reasons.